client/test/cypress/integration/admin/publications.spec.js

```diff
@@ -25,36 +26,34 @@ describe('Admin Publications', () => {
     cy.dataCy('new_publication_input')
       .type('{enter}');
     cy.dataCy('publications_list');
-    cy.dataCy('create_publication_notify').should('be.visible').should('have.class','bg-negative');
-    cy.injectAxe();
+    cy.dataCy('name_field_error').should('be.visible');
     cy.dataCy('new_publication_input')
       .type('Draft Publication from Cypress');
     cy.checkA11y();
   });
 
   it('prevents publication creation when the name exceeds the maximum length', () => {
-    const name_256_characters = '0123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345';
+    const name_256_characters = '01234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123450{enter}';
```

Comment:
The `{enter}` is already present on line 38 and the addition of the `0` character makes this a 257 character string. Either rename the variable to `name_257_characters` and change line 38 or revert this change as suggested.
```suggestion
    const name_256_characters = '0123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345';
```

Reply:
I need to investigate the underlying reason why I had to do this to get the test to pass.

Reply:
Updated.